### ISSUE TITLE
Cannot import ScalarRange

### ISSUE THREAD

**Issue by handenurc** [sdv-dev/SDV#1022]
Hi, I'm using SDV version of 0.15.0, I'm trying to train a CTGAN model for a single table data with minimum and maximum value constraints in every single column. I tried to use ScalarRange for that purpose, however I'm not able to import it properly:

`from sdv.constraints import ScalarRange` gives the error as such:

`ImportError: cannot import name 'ScalarRange' from 'sdv.constraints'`

However, I could import FixedCombinations which is another constraint that I've already used for the model.

If it is a version issue, in your documentation, it is not clearly stated that ScalarRange works only at specific versions. If it is not a version issue, how can I implement this constraint?

**Comment by npatki** [comment]
Hi @handenurc, nice to meet you!

This is a version issue, as [SDV 0.16.0](https://github.com/sdv-dev/SDV/releases/tag/v0.16.0) included a number of constraint updates. Is it possible for you to upgrade your project to the latest version? If not, I can find the old reference and share it here.

> If it is a version issue, in your documentation, it is not clearly stated that ScalarRange works only at specific versions. 

Thanks for the feedback -- agreed this is confusing! Our documentation is always up-to-date for the latest SDV version. We can add a notice. Note that our project is still in a pre-Alpha stage, so some small parts of the API & naming may be changing. As we mature, we expect to have more backwards-compatibility in the future.

FYI by default the CTGAN model learns the min and max values per column, only generating synthetic data that falls into the observed range. The relevant parameters in 0.15.0 are `min_value='auto'` and `max_value='auto'`. The `ScalarRange` constraint will not have an effect in this case.

**Comment by handenurc** [comment]
Thank you for your quick and detailed response! I will arrange myself according to this information. 

I'd like to ask one more question that occupies my mind. I have a single table data that has around 30 features and 60000 samples. With a trained model (say CTGAN), I generate two different sized dataset (say 1.000 and 100.000 samples). Do we have the knowledge that as the #of samples increase, the model generates more samples always distributed in between min and max values? Or does the model have the freedom to (i.e. 1000 sampled) move around some limited distribution range due to less #of samples? I mean, if we observe min and max values of the newly created dataset are not close to the original dataset (such as some missed/smoothed  outliers), is it related with being generated small #of samples (like 100 or 1000 samples)? Or is it the characteristic of the model?

**Comment by npatki** [comment]
No problem @handenurc! 

This is an interesting question that may require some more discussion. I have created a new issue for it in #1025 where I will provide some thoughts.

As for the original issue related to importing `ScalarRange` -- seems like we can close it since it was due to versioning? If there is anything more on the original `ScalarRange` topic, feel free to reply and I will reopen the issue.
